app/views/dashboard/_user_topics.html.erb

```diff
@@ -0,0 +1,60 @@
+<div class="card-view" style="margin-left: -13px;">
+  <% current_user.subscriptions(:tag).each do |tag| %>
+    <div class="card shadow-sm p-2 mb-3 bg-white rounded" >
+      <div class="card-header mt-2" style=" padding:0.8em; background-color: inherit; border:none;">
+        <h4 style="display: inline-block;"><%= tag.tagname %></h4>
+        <p style="display: inline-block; color: #808080;"><a style="text-decoration: underline; color: #808080;" href="/tag/<%= tag.tagname %>"><%= Tag.follower_count(tag.tagname) %> people</a> discussing</p>
+      </div>
+      <div class="card-body" style="padding:0.8em;">
+        <div class="node-list">
+          <% Tag.find_nodes_by_type(tag.tagname, type = 'note', limit = 3).each do |node|  %>
```

Comment:
In dashboard, we require all the nodes, `Comments`, `wikis`, `Questions` too I guess wiki too. I think this would be updated in profile page too. I'll update it for profile page.

Reply:
Okay! I'll add them as well. Thanks!

Reply:
Hi @CleverFool77! How do I add comments here, because we cannot associate comments with any topic?

Reply:
Shall we approach this the same way as on the profile page, by making a new page similar to `/comments`, but for `/comments/warren` for example -- all comments by an author? Then we can link there from the dashboard AND from the profile page.

Reply:
We could make it a pretty simple page, just a table like /comments maybe?